### PR TITLE
Stop running filesizes through size_format twice.

### LINKED ISSUE #1058: Incorrect File Sizes Recorded under Exclusions
Pretty sure this might be universal - have seen on every site I've looked at running BWP. 

When you go under exclusions, the files sizes are listed incorrectly. So if all the files together are 600 MB, and the uploads folder is actually 300 MB in size, it just shows 300 B in size. 

tools php 1

### PR DESCRIPTION
**Pull request by willmot**
Fixes #1058 issues with excludes filesizes showing incorrectly.

### CODE REVIEW

**Comment by dashaluna** [comment]
@willmot related WP bug? https://github.com/WordPress/WordPress/blob/master/wp-includes/functions.php#L262

File: admin/schedule-form-excludes.php [hunk 1]
```diff
@@ -8,10 +8,10 @@
 <div class="hmbkp-exclude-settings">
 
 	<h3>
-		<?php _e( 'Currently Excluded', 'backupwordpress' ); ?>
+		<?php esc_html_e( 'Currently Excluded', 'backupwordpress' ); ?>
 	</h3>
 
-	<p><?php _e( 'We automatically detect and ignore common <abbr title="Version Control Systems">VCS</abbr> folders and other backup plugin folders.', 'backupwordpress' ); ?></p>
+	<p><?php esc_html_e( 'We automatically detect and ignore common Version Control Systems folders and other backup plugin folders.', 'backupwordpress' ); ?></p>
```
Comment (dashaluna) [inline admin/schedule-form-excludes.php]: I've got rid of `abbr` here, I don' think there is much for the benefit of having it - I'd prefer "don't make me think" approach when it's spelt straight away. Also makes escaping easier ;)

**Comment by dashaluna** [comment]
As I've mentioned I will be fixing code that I work on to adhere to WP Coding Standards.

File: admin/schedule-form-excludes.php (hunk 1 above)
Comment (willmot) [inline admin/schedule-form-excludes.php]: 👍 agreed

**Comment by willmot** [comment]
> As I've mentioned I will be fixing code that I work on to adhere to WP Coding Standards.

👌

**Comment by dashaluna** [comment]
@pdewouters ready for review & merge

**Comment by willmot** [comment]
@dashaluna thanks for fixing up the unescaped translations as well 👍

**Pull request merged**